Snakemake/workflow/rules/ancestral_inference.smk

```diff
@@ -69,33 +69,50 @@ if config['ancestralAllele'] == None:
             vcfAlleles=rules.extract_snps_from_info.output
         conda: "HLab_tsinfer"
         output:
-            expand("../Project/AncestralAllele/Estsfs/EstSfs_Dict{{chunk}}.csv")
+           [f"../{Project}/AncestralAllele/Estsfs/" + x for x in expand("EstSfs_Dict{{chunk}}.csv", chunk = range(config['no_estsfs_chunks']))]
```

Comment:
In the ancestral script, I've changed the paths to point to the {Project}, instead of Project. But maybe everything needs to point to be workdir instead?